opencog/atoms/execution/Instantiator.cc

```diff
@@ -135,7 +135,7 @@ Handle Instantiator::walk_tree(const Handle& expr)
 		// If we are here, we found a variable. Look it up. Return a
 		// grounding if it has one, otherwise return the variable
 		// itself.
-		std::map<Handle,Handle>::const_iterator it = _vmap->find(expr);
+		HandleMap::const_iterator it = _vmap->find(expr);
```

Comment:
these kinds of changes make the code harder to read and harder to understand.  Why make them?